src/lib/loaders/loaders_with_authentication/gravity.ts

```diff
@@ -900,6 +900,14 @@ export default (accessToken, userID, opts) => {
       {},
       { headers: true }
     ),
+    partnerContactLoader: gravityLoader<
+      any,
+      { partnerId: string; contactId: string }
+    >(
+      ({ partnerId, contactId }) => `partner/${partnerId}/contact/${contactId}`,
+      {},
```

Comment:
[Hitting this guy](https://github.com/artsy/gravity/blob/main/app/api/v1/partners_contacts_endpoint.rb#L67)